apps/cookbook/src/app/examples/header-example/examples/subtitles.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+
+const config = {
+  selector: 'cookbook-header-example-subtitle-list',
+  template: `<kirby-header [title]="'Title'" [subtitle1]="['Mrs. Lady Like Tiff Tuff Escargoon Tokkori ', '&', 'Mr. Chef Kawasaki Tokkori']">
```

Comment:
```suggestion
  template: `<kirby-header [title]="'Title'" [subtitle1]="['Mrs. Lady Like Tiff Tuff Escargoon Tokkori', '&', 'Mr. Chef Kawasaki Tokkori']">
```

Consider removing this space to avoid confusion - consumers might think they need to supply a space